toptal/picasso
# feat(page-header): add logoLink to PageHeader component

fix #354

### Description

Adds `logoLink` to `Page.Header`. Example usage:
```
<PageHeader
    title='Something'
    logoLink={<Link href='https://www.toptal.com' />}
/>
```

### How to test

- Use storybook and check [a new example](https://picasso.toptal.net/add-logo-href-to-page-header/?path=/story/layout-folder--page#link) for `Page.Header`

### Screenshots

No visual changes

### Review

- [x] Ensure that visuals specs are green [See the documentation](/README.md#fixing-broken-visual-tests-inside-a-pr)
